core/src/templates/components/card/card.twig

```diff
@@ -3,78 +3,94 @@
  * @file
  * Card Component.
  *
- * Basic building block component with two variants
- *   - Horizontal: An image beside text content
+ * Basic building block component with the following variants:
  *   - Vertical (default): An image above text content
+ *   - Horizontal: An image beside text content
+ *   - Link: Whole card is a link with no button or call to action link
  *
  * Available variables:
  * - attributes: For additional HTML attributes not already provided.
- * - modifier_class: Additional css classes to change look and behaviour.
- * -- su-card--horizontal
+ * - modifier_class: Additional CSS classes to change look and behaviour.
  * - card_link: The href value for the card. A single card is only allowed to link to one URL.
  * - card_image: A full HTML entity for an image.
- * - card_super_headline: Text heading
+ * - card_super_headline: Text super headline. Often used for metadata such as category.
  * - card_headline: Text heading
  * - card_body: An open variable for anything. Generally this would be for
  *     WYSIWYG editor contents.
- * - card_cta_attributes: Any html attributes on the cta link that need to be
+ * - card_cta_attributes: Any html attributes on the link that need to be
  *     added. For example: _target.
  * - card_cta_label: The clickable text for the CTA link
  * - card_button_label: The text label of the button component.
  */
 #}
-<div{{ attributes }} class="su-card {{ modifier_class }}">
 
-  {# Card Image #}
-  {% if card_image is not empty -%}
-    {{ card_image }}
+{# For variant where whole card is a link, change the outer wrapper from <div> to <a> #}
+{% if "su-card--link" in modifier_class %}
+  {% set allow_links = false %}
+{% else %}
+  {% set allow_links = true %}
+{%- endif -%}
+
+<div {{ attributes }} class="su-card {{ modifier_class }}">
+  {%- if allow_links == false %}
+    <a href="{{ card_link }}" {{ card_cta_attributes }}>
   {%- endif -%}
 
-  <div class="su-card__contents">
+    {# Card Image #}
+    {% if card_image is not empty -%}
+      {{ card_image }}
+    {%- endif -%}
 
-  {# Card Super Headline #}
-  {%- block block_card_super_headline %}
-    {% if card_super_headline is not empty %}
-      <h4>{{ card_super_headline }}</h4>
-    {% endif -%}
-  {% endblock -%}
+    <div class="su-card__contents">
 
-  {# Card Headline #}
-  {%- block block_card_headline  %}
-    {%- if card_headline is not empty %}
-      <h3>{{ card_headline }}</h3>
-    {% endif -%}
-  {% endblock -%}
+    {# Card Super Headline #}
+    {%- block block_card_super_headline %}
+      {% if card_super_headline is not empty %}
+        <span>{{ card_super_headline }}</span>
+      {% endif -%}
+    {% endblock -%}
 
-  {# An open field for body content #}
-  {{ card_body }}
+    {# Card Headline #}
+    {%- block block_card_headline  %}
+      {%- if card_headline is not empty %}
+        <h3>{{ card_headline }}</h3>
```

Comment:
What about making this an h2?

Reply:
This is probably site dependent but it seems a little more common as h3? Since a lot of page headers are h2, so card titles should be at least one step lower? One example is the Stanford homepage. I could be convinced though 😄

Reply:
`<h3>` leaves us more vulnerable to validation / accessibility warnings about skipping levels. We can be reasonably certain any page a card is on will have an `<h1>`, but it's reasonably likely that a page doesn't have an `<h2>`. Also, if we're changing the container to `<article>` per #421, `<h2>` might make more sense. So I guess I have a slight preference for `<h2>`. Which, for the record, is the opposite of my inclination when I started typing this comment. I swayed myself! 🙀

Reply:
Haha....fine. h2 it is.

Reply:
Headline has been changed to h2